amethyst_network/src/connection.rs

```diff
@@ -100,3 +161,59 @@ impl Default for NetIdentity {
 impl Component for NetIdentity {
     type Storage = VecStorage<NetIdentity>;
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::connection::NetConnection;
+    use crate::net_event::NetEvent;
+
+    #[test]
+    fn can_register_reader() {
```

Comment:
Hrm. Seems to be missing an assert

Reply:
I am going to remove this test, it can't fail and other tests are also using the register function.